docs/UserGuide.md

```diff
@@ -77,156 +105,254 @@ Format: `help`
 ---
 ### Adding an applicant: `add`
 
-Adds a new applicant to be tracked, by adding their name, phone number, email, the job they have applied for, round of job application, along with a variable number of skills.
+Allows you to add a new applicant to be tracked with the following details:
+* Name
+* Phone Number
+* Email
+* The job they've applied for
+* Round of job application
+* Particular skills they may have
+
+<br>
 
 Format:
 ```
 add n/NAME p/PHONE_NUMBER e/EMAIL j/JOB r/ROUND [s/SKILLS]...
 ```
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of skills (including 0)
+A person can have any number of skills (including none at all)
```

Comment:
Is it better to state it as an applicant?
```suggestion
An applicant can have any number of skills (including none at all)
```

Reply:
Good spot, thanks!